File: Programming basics/Homework-Console-Input-Output/Print Company Information/PrintCompanyInformation.cs
using System;
class PrintCompanyInformation
{
    static void Main()
    {
        Console.Write("Company name: ");
        string companyName = Console.ReadLine();

        Console.Write("Company address: ");
        string companyAddress = Console.ReadLine();

        Console.Write("Phone number: ");
        string phoneNumber = Console.ReadLine();

        Console.Write("Fax number: ");
        string faxNumber = Console.ReadLine();

        Console.Write("Web site: ");
        string website = Console.ReadLine();

        Console.Write("Manager first name: ");
        string managerFirstName = Console.ReadLine();

        Console.Write("Manager last name: ");
        string managerLastName = Console.ReadLine();

        Console.Write("Manager age: ");
        int managerAge = int.Parse(Console.ReadLine());

        Console.Write("Manager phone number: ");
        string managerPhoneNumber = Console.ReadLine();

        Console.WriteLine("---------------------");

        Console.WriteLine(companyName);
        Console.WriteLine(companyAddress);
        Console.WriteLine("Tel. " + phoneNumber);
        if (faxNumber == "")
        {
            Console.WriteLine("Fax: (no fax)");
        }
        else
        {
            Console.WriteLine("Fax:" + faxNumber);
        }
        Console.WriteLine("Web site:" + website);
        Console.WriteLine("Manager: {0} {1} (age: {2}, tel. {3})", managerFirstName, managerLastName, managerAge, managerAge);



    }
}

// A company has name, address, phone number, fax number, web site and manager. The manager has first name, last name,
// age and a phone number. Write a program that reads the information about a company and its manager and prints it back on the console.

// program	        user
// Company name:	Software University
// Company address:	15-18 Tintyava, Sofia
// Phone number:	+359 899 55 55 92
// Fax number:	
// Web site:	    http://softuni.bg
// Manager first name:	Svetlin
// Manager last name:	Nakov
// Manager age:	25
// Manager phone:	    +359 2 981 981

// Software University
// Address: 26 V. Kanchev, Sofia
// Tel. +359 899 55 55 92
// Fax: (no fax)
// Web site: http://softuni.bg
// Manager: Svetlin Nakov (age: 25, tel. +359 2 981 981)	

File: 4.1 C Programming/Homework 3 Formatted Input Output/PrintCompanyInformation.c
#include <stdio.h>
#include <string.h>
#define STRING_SIZE 30

int main() 
{
    char companyName[STRING_SIZE],
            companyAddress[STRING_SIZE],
            companyPhoneNumber[STRING_SIZE],
            companyFaxNumber[STRING_SIZE],
            companyWebSite[STRING_SIZE],
            managerFirstName[STRING_SIZE],
            managerLastName[STRING_SIZE],
            managerPhoneNumber[STRING_SIZE],
            managerAge[STRING_SIZE];
    
    printf("Company name: ");
    fgets(companyName, STRING_SIZE, stdin);
    printf("Company address: ");
    fgets(companyAddress, STRING_SIZE, stdin);
    printf("Phone number: ");
    fgets(companyPhoneNumber, STRING_SIZE, stdin);
    printf("Fax number: ");
    fgets(companyFaxNumber, STRING_SIZE, stdin);
    if (companyFaxNumber[0] == '\n')
    {
        strncpy(companyFaxNumber, "(no fax)\n", 9);
    }
    
    printf("Web site: ");
    fgets(companyWebSite, STRING_SIZE, stdin);
    printf("Manager first name: ");
    fgets(managerFirstName, STRING_SIZE, stdin);
    size_t firstNameLength = strlen(managerFirstName) - 1;
    if (managerFirstName[firstNameLength] == '\n')
    {
        managerFirstName[firstNameLength] = '\0';
    }
    
    printf("Manager last name: ");
    fgets(managerLastName, STRING_SIZE, stdin);
    size_t lastNameLength = strlen(managerLastName) - 1;
    if (managerLastName[lastNameLength] == '\n')
    {
        managerLastName[lastNameLength] = '\0';
    }
    
    printf("Manager age: ");
    fgets(managerAge, STRING_SIZE, stdin);
    size_t ageLength = strlen(managerAge) - 1;
    if (managerAge[ageLength] == '\n')
    {
        managerAge[ageLength] = '\0';
    }
    
    printf("Manager phone: ");
    fgets(managerPhoneNumber, STRING_SIZE, stdin);   
    size_t phoneLength = strlen(managerPhoneNumber) - 1;
    if (managerPhoneNumber[phoneLength] == '\n')
    {
        managerPhoneNumber[phoneLength] = '\0';
    }
    
    printf("%sAddress: %sTel. %sFax: %sWeb site: %sManager: %s %s (age: %s, tel. %s)\n",
            companyName,
            companyAddress,
            companyPhoneNumber,
            companyFaxNumber,
            companyWebSite,
            managerFirstName,
            managerLastName,
            managerAge,
            managerPhoneNumber);    
    
    return 0;
}